docs/reference/pages/panels.rst

```diff
@@ -243,6 +243,24 @@ SnippetChooserPanel
     See :ref:`snippets` for more information.
 
 
+HelpPanel
+---------
+
+.. module:: wagtail.admin.edit_handlers
+
+.. class:: HelpPanel(content, template)
+
+    .. attribute:: HelpPanel.content
+
+        This is string HTML that you can pass in directly to render inside the HelpPanel
+
+    .. attribute:: HelpPanel.template
+
+        Alternatively, you can pass in a template name. Specifying the template will override the content attribute.
```

Comment:
I would change this to:
`Path to a template rendering the full panel HTML.`

No need to specify how the template itself work, since there is a default template for help panel, people can just look at it to see how it works.